src/poudriered/poudriered.c

```diff
@@ -897,6 +897,10 @@ main(int argc, char **argv)
 
 	const ucl_object_t *sock_path_o, *pidfile_path_o, *foreground_o;
 
+	if (argc == 2 && strcmp(argv[1], "-f") == 0) {
+		foreground = 1;
```

Comment:
let's use `true` here. this *is* a boolean variable

Reply:
Well, that is sloppy isn't it, I'm sorry!  I'll fix it.